packages/shared/src/types/metadata.ts

```diff
@@ -0,0 +1,30 @@
+import { ArenaMatchEndInfo, ArenaMatchStartInfo, CombatResult, WowVersion } from '@wowarenalogs/parser';
+
+export interface IMetadata {
```

Comment:
this looks very similar to the combat object - what's different that requires another interface?

Reply:
It's more or less the combat stub - I need to take another attempt at using the stub types directly but we need to make sure we don't write the entire parsed combat into the video metadata. This would make reading the metadata files much slower and we need to be able to scan them quickly in the current implementation